spec/hobbit_spec.rb

```diff
@@ -58,21 +58,36 @@
     expect(hobbit.adult?).to be true
   end
 
-  xit 'is old at the age of 101' do
-    # create a hobbit
+  it 'is old at the age of 101' do
+    hobbit = Hobbit.new('Stemmle')
+
+    101.times do
+      hobbit.celebrate_birthday
+    end
     # have hobbit age 101 years
+    expect(hobbit.old_hobbit?).to be true
     # check that hobbit.old? returns true
+    hobbit.celebrate_birthday
+
+    expect(hobbit.old_hobbit?).to be true
   end
```

Comment:
@Taamir-Ransome most of the commits so far have been atomic and related well to the commit message. In this case, adding a test for age in this commit has nothing to do with the method for ring. See if you can use lazygit to move this work to the proper commit